_posts/2022-05-15-homebrew-security-audit.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Security Audit
+author: MikeMcQuaid
+---
+
+Homebrew has had a paid security audit and addressed all flagged issues. This blog post has been a long time coming, apologies for the delay.
+
+Here's an overview of the timescale:
+
+- 11th June 2020: the [Mozilla Open Source program (MOSS)](https://www.mozilla.org/en-US/moss/) reaches out to Homebrew as we were nominated for a paid, sponsored security audit by [Radically Open Security (ROS)](https://radicallyopensecurity.com/)
+- 11th June 2020: Homebrew is introduced to ROS and provides the main areas of focus:
```

Comment:
Subject/object confusion here. If Homebrew provided the main areas of focus, it should be the subject, e.g. `Homebrew meets with ROS and…`